chromium_src/chrome/browser/permissions/permission_uma_util.cc

```diff
@@ -3,37 +3,45 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this file,
  * You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-#include "chrome/browser/permissions/permission_util.h"
+#include "components/permissions/permission_util.h"
 
-#include "chrome/browser/permissions/permission_request.h"
 #include "components/content_settings/core/common/content_settings_types.h"
+#include "components/permissions/permission_request.h"
 
 namespace {
 
-std::string GetPermissionRequestString_ChromiumImpl(PermissionRequestType type);
+std::string GetPermissionRequestString_ChromiumImpl(
+    permissions::PermissionRequestType type);
 void BraveRecordPermissionAction(ContentSettingsType permission,
                                  bool secure_origin,
-                                 PermissionAction action);
+                                 permissions::PermissionAction action);
 
-std::string GetPermissionRequestString(PermissionRequestType type) {
-  if (type == PermissionRequestType::PERMISSION_AUTOPLAY)
+std::string GetPermissionRequestString(
+    permissions::PermissionRequestType type) {
+  if (type == permissions::PermissionRequestType::PERMISSION_AUTOPLAY)
     return "Autoplay";
-  if (type == PermissionRequestType::PERMISSION_WIDEVINE)
+  if (type == permissions::PermissionRequestType::PERMISSION_WIDEVINE)
     return "Widevine";
-  if (type == PermissionRequestType::PERMISSION_WALLET)
+  if (type == permissions::PermissionRequestType::PERMISSION_WALLET)
     return "Wallet";
   return GetPermissionRequestString_ChromiumImpl(type);
 }
 
 }  // namespace
 
+#define BRAVE_PERMISSIONUMAUTIL_RECORDPERMISSIONACTION              \
```

Comment:
I was trying to find where this was used; looks like it was patched in before this commit? (via https://github.com/brave/brave-core/commit/fdfe5a85f772d837c27bee4e563cd431f84eb103)